build/follow-me/render.php

```diff
@@ -7,29 +7,31 @@
 
 use Activitypub\Blocks;
 use Activitypub\Collection\Actors;
+use Activitypub\Collection\Followers;
 
 /* @var array $attributes Block attributes. */
 $attributes = wp_parse_args( $attributes );
 
+/* @var WP_Block $block Parsed block.*/
+$block = $block ?? null;
+
+/* @var string $content Inner blocks content. */
+$content = $content ?? '';
+
 // Get the user ID from the selected user attribute.
-$selected_user = $attributes['selectedUser'] ?? 'site';
-$user_id       = Blocks::get_user_id( $selected_user );
-$button_only   = $attributes['buttonOnly'] ?? false;
+$user_id = Blocks::get_user_id( $attributes['selectedUser'] ?? 'site' );
```

Comment:
I would prefer to be consistant with the rest of the code and use `0` for the site user. We have so many different wordings, that it gets quite complicated to see if it is the same or not. Aside from that, in PHP we call it blog-user. No strong opinion about the name, but it should be consistant.